includes/options/class-amp-setup-wizard-submenu.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * AMP setup wizard.
+ *
+ * @package AMP
+ * @since @todo NEW_ONBOARDING_RELEASE_VERSION
+ */
+
+/**
+ * AMP setup wizard class.
+ *
+ * @since @todo NEW_ONBOARDING_RELEASE_VERSION
+ */
+class AMP_Setup_Wizard_Submenu {
```

Comment:
Classes should default to `final`, unless we have a specific use case we want to open up extensibility for.
```suggestion
final class AMP_Setup_Wizard_Submenu {
```